### PR TITLE
clang-tidy: validate casing of private members

### PR DESCRIPTION
**Pull request by derekargueta**
Description: standardize private class members as `lower_case_`.
Risk Level: low
Testing: existing
Docs Changes: N/A
Release Notes: N/A

Signed-off-by: Derek Argueta <dereka@pinterest.com>

### CODE REVIEW

File: source/common/config/grpc_stream.h [hunk 1]
```diff
@@ -129,8 +129,8 @@ class GrpcStream : public Grpc::AsyncStreamCallbacks<ResponseProto>,
   GrpcStreamCallbacks<ResponseProto>* const callbacks_;
 
   // TODO(htuch): Make this configurable or some static.
-  const uint32_t RETRY_INITIAL_DELAY_MS = 500;
-  const uint32_t RETRY_MAX_DELAY_MS = 30000; // Do not cross more than 30s
+  const uint32_t retry_initial_delay_ms_ = 500;
```
Comment (lizan) [inline source/common/config/grpc_stream.h]: make this static for now and keep the case? this is constant right?

Comment (derekargueta) [reply source/common/config/grpc_stream.h]: done

File: source/common/runtime/runtime_impl.cc [hunk 2]
```diff
@@ -378,8 +378,8 @@ DiskLayer::DiskLayer(absl::string_view name, const std::string& path, Api::Api&
 void DiskLayer::walkDirectory(const std::string& path, const std::string& prefix, uint32_t depth,
                               Api::Api& api) {
   ENVOY_LOG(debug, "walking directory: {}", path);
-  if (depth > MaxWalkDepth) {
```
Comment (jmarantz) [inline source/common/runtime/runtime_impl.cc]: why this change? ThisStyle for constants is recommended in the style guide. ALL_CAPS is OK but discouraged due to possibility of clashing with macros.

Comment (derekargueta) [reply source/common/runtime/runtime_impl.cc]: I was following the stil on lines 132-133 of `source/common/config/grpc_stream.h` and conforming the rest, but I can do it the other way around.

Comment (lizan) [reply source/common/runtime/runtime_impl.cc]: +1 to ThisStyle

Comment (derekargueta) [reply source/common/runtime/runtime_impl.cc]: updated, along with some ODR-use workarounds which I imagine was why these weren't `static constexpr` previously.

(I'd personally vote for `kThisStyle` to distinguish from types which use `ThisStyle`, but we can chat more when I get around to `StaticConstantCase` and `ConstexprVariableCase`)

File: source/common/config/grpc_stream.h [hunk 3]
```diff
@@ -35,8 +35,12 @@ class GrpcStream : public Grpc::AsyncStreamCallbacks<ResponseProto>,
           rate_limit_settings.max_tokens_, time_source_, rate_limit_settings.fill_rate_);
       drain_request_timer_ = dispatcher.createTimer([this]() { callbacks_->onWriteable(); });
     }
-    backoff_strategy_ = std::make_unique<JitteredBackOffStrategy>(RETRY_INITIAL_DELAY_MS,
-                                                                  RETRY_MAX_DELAY_MS, random_);
+
+    // TODO(htuch): Make this configurable or some static.
```
Comment (lizan) [inline source/common/config/grpc_stream.h]: remove this TODO or the latter part?
